### ISSUE TITLE
Error calling `avworkspace_configuration_inputs`

### ISSUE THREAD

**Issue by amstilp** [Bioconductor/AnVIL#80]
I'm trying to submit a workflow job from an Rstudio session on AnVIL using the Bioconductor package, following the [vignette](https://bioconductor.org/packages/release/bioc/vignettes/AnVIL/inst/doc/RunningWorkflow.html).

```
avworkflow("primed-cc/validate_genotype_model")
config <- avworkflow_configuration_get()
config

inputs <- avworkflow_configuration_inputs(config) # This doesn't work
```

The error I get is:
```
Error in .avstop_for_status(response) : 
  argument "op" is missing, with no default
```

I tried tracing the code back and it looks like it might be happening on [line 106](https://github.com/Bioconductor/AnVIL/blob/e7f43e68912dec4c835adc01d7f59315c43cb74f/R/avworkflow_configuration.R#L106) of the avworkflow_configuration.R source file, which does not pass the `op` argument to `.avstop_for_status`.

---

Here's my sessionInfo:
```
> sessionInfo()
R version 4.3.0 (2023-04-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 22.04.1 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.10.0 
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.10.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Etc/UTC
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

other attached packages:
[1] AnVIL_1.12.1 dplyr_1.1.2 

loaded via a namespace (and not attached):
 [1] miniUI_0.1.1.1       jsonlite_1.8.4       futile.logger_1.4.3 
 [4] compiler_4.3.0       promises_1.2.0.1     tidyselect_1.2.0    
 [7] Rcpp_1.0.10          parallel_4.3.0       tidyr_1.3.0         
[10] later_1.3.0          yaml_2.3.7           fastmap_1.1.1       
[13] mime_0.12            R6_2.5.1             generics_0.1.3      
[16] curl_5.0.0           htmlwidgets_1.6.2    rapiclient_0.1.3    
[19] tibble_3.2.1         shiny_1.7.4          pillar_1.9.0        
[22] rlang_1.1.1          utf8_1.2.3           DT_0.28             
[25] httpuv_1.6.9         cli_3.6.1            magrittr_2.0.3      
[28] formatR_1.14         futile.options_1.0.1 digest_0.6.31       
[31] rstudioapi_0.14      xtable_1.8-4         lifecycle_1.0.3     
[34] vctrs_0.6.2          glue_1.6.2           lambda.r_1.2.4      
[37] fansi_1.0.4          purrr_1.0.1          httr_1.4.6          
[40] tools_4.3.0          pkgconfig_2.0.3      ellipsis_0.3.2      
[43] htmltools_0.5.5     

```

**Comment by amstilp** [comment]
Also adding the contents of the config object that's returned by `avworkflow_configuration_get()`:

```
> config
List of 8
 $ deleted            : logi FALSE
 $ inputs             : Named list()
 $ methodConfigVersion: int 1
 $ methodRepoMethod   :List of 4
  ..$ methodUri    : chr "dockstore://github.com%2FUW-GAC%2Fprimed-file-checks%2Fvalidate_genotype_model/combined_checks"
  ..$ sourceRepo   : chr "dockstore"
  ..$ methodPath   : chr "github.com/UW-GAC/primed-file-checks/validate_genotype_model"
  ..$ methodVersion: chr "combined_checks"
 $ name               : chr "validate_genotype_model"
 $ namespace          : chr "primed-cc"
 $ outputs            : Named list()
 $ prerequisites      : Named list()
 - attr(*, "class")= chr [1:2] "avworkflow_configuration" "list"
```

**Comment by mtmorgan** [comment]
Thanks @amstilp this is fixed in version 1.12.4 which can be installed now with `remotes::install_github("Bioconductor/AnVIL", ref = "RELEASE_3_17")`.

This fixes the immediate bug; it might be that it just enables reporting of an error that may itself not be obvious; feel free to open a second issue or continue here...

**Comment by amstilp** [comment]
Thanks, I installed the new version and it gives a much more informative error message - now I'm off to track down the problem with my workflow on AnVIL (not related to the Bicoonductor AnVIL package).
